.evergreen/config.yml

```diff
@@ -0,0 +1,629 @@
+# When a task that used to pass starts to fail
```

Comment:
what is the difference between this file and `.evg.yml`?

Reply:
ack `.evg.yml` was accidentally commited. Removing now, and disregarding review from that file.

Reply:
ok, then transplant all my comments from `.evg.yml` to `evergreen/config.yml`

Reply:
honestly I already squashed a whole bunch of commits here, I'm inclined to leave it as is